package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "jest -c jest.config.js --watch",
+    "test": "jest -c jest.config.js",
```

Comment:
The --watch flag causes issues with GitHub actions, as if there are no changes to the test in the commit, the step will be hanging saying a "No tests found." message like the one you saw i.e.
![image](https://user-images.githubusercontent.com/43914992/183255905-7903ecb3-ed9b-46c9-856d-e6ebf0d76595.png)

Running it without watch ensures all the tests always run